src/containers/Moods.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Controls from '../components/controls/Controls';
+import Face from '../components/face/Face';
+import Timer from '../components/timer/Timer';
+import styles from './Moods.css';
+import { DRINK_COFFEE, EAT_SNACK, TAKE_NAP, STUDY, RESET_GAME, saveGame } from '../actions/moodsActions';
+import { getCoffees, getSnacks, getNaps, getStudies, getFace } from '../selectors/moodsSelectors';
+import History from '../components/History/History';
+
+const actions = [
+  { name: DRINK_COFFEE, text: 'Drink Coffee', stateName: 'coffees' },
+  { name: EAT_SNACK, text: 'Snack', stateName: 'snacks' },
+  { name: TAKE_NAP, text: 'Nap', stateName: 'naps' },
+  { name: STUDY, text: 'Study', stateName: 'studies' },
+];
+
+const Moods = ({ count, face, actions, handleSelection, handleReset, handleSaveGame, history }) => {
```

Comment:
you don't need to pass actions in here if by actions you are referring to the array on the above lines.